app/controllers/user_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UserController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end

```

Comment:
- [ ] Please notice that In Ruby on Rails, the naming convention for controllers is to use the plural form of the resource name. For example, if you have a resource named "user", the corresponding controller should be named "UsersController". Additionally, the controller class should be defined in a file with the same name as the class, but with underscores instead of camel case. So in this example, the controller should be defined in a file named "users_controller.rb". please fix this
currently, we are getting the error :point_down: 
![image](https://user-images.githubusercontent.com/97521557/229861790-e4120b56-82c0-45fe-bf6d-f2b869ed653b.png)